packages/commands-http/src/request.ts

```diff
@@ -0,0 +1,84 @@
+import { Command } from '@seagull/commands'
+import fetch, { Response as FetchResponse } from 'node-fetch'
+import { Http, RequestConfig } from './index'
+import SeedStorage from './seedStorage'
+
+/**
+ * Http request command.
+ */
+export class Request<T> extends Command<T> {
+  executeConnected = this.executeCloud
+  executeEdge = this.executeCloud
+
+  private seed: SeedStorage<T>
+
+  /**
+   * Creates an http request command.
+   * @param config Http request command configuration.
+   */
+  constructor(public config: RequestConfig) {
+    super()
+    if (!config) {
+      throw new Error('HttpCommand: You have to specify a config!')
+    }
+    if (!this.config.url) {
+      throw new Error('HttpCommand: The url in the config is not specified!')
+    }
+    this.config = config
+    this.seed = SeedStorage.createByRequest(config.url, config.init)
+  }
+
+  async executeCloud(): Promise<T> {
+    const response = await fetch(this.config.url, this.config.init)
+    switch (this.config.parseBody) {
+      case 'xml':
+        throw new Error('not implemented')
```

Comment:
Can't we return `response.text()` or `textConverted()` instead of raising an error?

Reply:
This is a placeholder for a xml parsing implementation, which we have within iris (i think) right now. So it'll probably be needed at some point and this might allow the developer to find the suggested point in code to implement this.